frontend/src/components/common/Resource/ResourceTable.tsx

```diff
@@ -12,8 +13,12 @@ type SimpleTableColumn = SimpleTableProps['columns'][number];
 type ColumnType = 'age' | 'name' | 'namespace' | 'type' | 'kind';
 
 export interface ResourceTableProps extends Omit<SimpleTableProps, 'columns'> {
-  /** The columns to be rendered, like used in SimpleTable, or by name */
+  /** The columns to be rendered, like used in SimpleTable, or by name. */
   columns: (SimpleTableColumn | ColumnType)[];
+  /** ID for the table. Will be used by plugins to identify this table. */
+  id?: string;
```

Comment:
Can you please add the naming convention here?

Reply:
Will do.